react-ebay-network-aware-code-splitting/src/components/CenterPanelInternal/ProductSummary/LeftSummary/index.js

```diff
@@ -0,0 +1,84 @@
+
+import React from 'react';
+
+import PropertyField from '../../PropertyField';
+import {
+  ConditionPropertyValue,
+  USSizePropertyValue,
+  QuantityPropertyValue,
+  ShippingPropertyValue,
+  DeliveryPropertyValue,
+  PaymentsPropertyValue,
+  ReturnsPropertyValue
+} from './assemblies';
```

Comment:
Could we give assemblies a more descriptive name?

Reply:
Yes, it sounds less descriptive as they are just for the purpose of demo elements.
Do you have some thing in your mind?

I'd suggest `SubComponents` or something.